ttax/ops.py

```diff
@@ -179,6 +183,9 @@ def add(tt_a, tt_b):
   Raises
     ValueError if the arguments shapes do not coincide.
   """
+  tt_a = tt1.tt if isinstance(tt1, WrappedTT) else tt1
```

Comment:
Let's put this logic inside wrapped TT? So that all wrapped logic is concentrated there.